moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -465,19 +468,71 @@ class OnlineRobot : public Robot
 
 
   /**
+   * @brief execute a given trajectory
+   * @param traj_name - name of trajectory to be executed (must be filled with
+   * prior calls to "addTrajPoint".
+   * @param goal - Joint trajectory goal which is a known 'Plan'
+   * @param collision_check - bool to turn check for collision on\off
+   * @throws <moveit_simple::ExecutionFailureException> (Execution failure)
+   * @throws <moveit_simple::IKFailException> (Conversion to joint trajectory failed)
+   * @throws <std::invalid_argument> (Trajectory "traj_name" not found)
+   * @throws <moveit_simple::CollisionDetected> (One of interpolated point is
+   * in Collision with itself or environment)
+   */
+  void executeKnownPlan(const std::string traj_name, 
```

Comment:
As opposed to an unknown plan?  The method signature should be different for this method, allowing you to use the same `execute` name.

Reply:
ok, I've changed the signature.